inc/cachify.class.php

```diff
@@ -641,7 +641,7 @@ public static function add_dashboard_count( $items = array() )
 
 		/* Right now item */
 		$items[] = sprintf(
-			'<a href="%s" class="cachify-icon cachify-icon--%s" title="%s: %s">%s esc_html__( 'Cache', 'cachify' )</a>',
+			'<a href="%s" class="cachify-icon cachify-icon--%s" title="%s: %s">%s Cache</a>',
```

Comment:
Maybe we should keep it translateable?

'<a href="%s" class="cachify-icon cachify-icon--%s" title="%s: %s">%s ' . esc_html__( 'Cache', 'cachify' ) . '</a>',
`